custom_components/audiconnect/dashboard.py

```diff
@@ -390,178 +393,302 @@ def create_instruments():
         TripData(attr="longterm_reset", name="LongTerm Trip User Reset"),
         Lock(),
         Preheater(),
-        Sensor(attr="model", name="Model", icon="mdi:car-info", unit=None),
-        Sensor(attr="mileage", name="Mileage", icon="mdi:speedometer", unit="km"),
+        Sensor(
+            attr="model",
+            name="Model",
+            icon="mdi:car-info",
+            unit=None,
+            state_class=None,
+            device_class=None,
+        ),
+        Sensor(
+            attr="mileage",
+            name="Mileage",
+            icon="mdi:speedometer",
+            unit="km",
+            state_class="total_increasing",
+            device_class=None,
+        ),
         Sensor(
             attr="service_adblue_distance",
             name="AdBlue range",
             icon="mdi:gas-station",
             unit="km",
+            state_class=None,
+            device_class=None,
+        ),
+        Sensor(
+            attr="range",
+            name="Range",
+            icon="mdi:gas-station",
+            unit="km",
+            state_class=None,
+            device_class=None,
         ),
-        Sensor(attr="range", name="Range", icon="mdi:gas-station", unit="km"),
         Sensor(
             attr="hybrid_range",
             name="hybrid Range",
             icon="mdi:gas-station-outline",
             unit="km",
+            state_class=None,

```

Comment:
Any reason to add None when it defaults to this?   Feels to me like longer code.

Reply:
I added the defaults after the fact. which caused it to be redundant. I'll clean those up.